unified-installer/astra-unified-installer.sh

```diff
@@ -2350,6 +2416,7 @@ step_apply_resources() {
     # Apply operator resources
     logdebug "apply operator resources"
     local output=""
+    local captured_err=""
```

Comment:
This variable declaration was missing, so the call 3 lines down was putting the value in a global variable instead.